### DISCUSSION TITLE
column level lineage from transforms/sparkSQL

### DISCUSSION THREAD

**Discussion by rsaw4** [AbsaOSS/spline#964] [Q&A]
Does spline show column level lineage from transforms/sparkSQL?

**Comment by wajda** [comment]
Yes it does. However the current UI (0.6) can only show the attribute-level lineage in the scope of one job. In the next UI we'll add ability to display it end-to-end across jobs.

**Comment by wajda** [comment]
Open the execution plan details, find your column on the left hand side or in the Input/Output schema of a selected operation, and click it. The column lineage will be highlighted on the operations graph. 

![image](https://user-images.githubusercontent.com/795479/122730506-92764680-d27a-11eb-859e-1145cf6b51b4.png)

Clicking on the Lineage or Impact details in the popup will show you the corresponding part of the attribute dependency graph.
